specification/servicelinker/resource-manager/readme.md

```diff
@@ -67,6 +67,15 @@ input-file:
   - Microsoft.ServiceLinker/preview/2022-11-01-preview/servicelinker.json
 ```
 
+### Tag: package-2023-04-01-preview
```

Comment:
add tag menas to generate SDK for the version. We onlyy plan to release python SDK for Azure CLI, right? If yes, could you pls specify language following readme syntax?